Neos.Neos/Classes/Fusion/Helper/NodeHelper.php

```diff
@@ -95,18 +95,6 @@ public function labelForNode(Node $node): NodeLabelToken
         return new NodeLabelToken($node);
     }
 
-    /**
-     * @param Node $node
-     * @return bool
-     * @deprecated Remove before Neos 9.0 !!! Use ${userInterfaceMode.isEdit || userInterfaceMode.isPreview} instead
```

Comment:
Wasn't it replaced by `renderingMode.isEdit` and `renderingMode.isPreview`? Should that be corrected in the merge request description?